action.yml

```
@@ -79,72 +83,20 @@ runs:
         aws_secret_access_key: ${{ inputs.aws_secret_access_key }}
         aws_region: ${{ inputs.aws_region }}
 
-    - name: Render and deploy preflight task definition to Amazon ECS
-      id: deploy-preflight-task-definition
-      uses: moneymeets/action-ecs-deploy/custom-deploy-steps/create-task-definition@master
+    - name: Deploy task definition and run preflight
+      id: deploy-definition-and-run-preflight
```

Probably _task_ definition missing in the name here. How about
```suggestion
      id: deploy-preflight-task-definition-and-run-task
```
?